recipes/tracy/all/conanfile.py

```diff
@@ -0,0 +1,95 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class TracyConan(ConanFile):
+    name = "tracy"
+    description = "C++ frame profiler"
+    topics = ("tracy", "profiler", "performance", "gamedev")
+    homepage = "https://github.com/wolfpld/tracy"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = ["BSD 3-clause"]
```

Comment:
```suggestion
    license = ["BSD-3-Clause"]
```

Follow the official short name: https://spdx.org/licenses/BSD-3-Clause.html

Reply:
Fixed